### ISSUE TITLE
Use timestamp to kick added users

### ISSUE THREAD

**Issue by ensiform** [Razish/node-jkpug#2]
Make use of the added timestamp to kick inactive users or look at discord/irc idle times as well.

**Comment by Razish** [comment]
My plan was to avoid having timers, but refresh the list of pending clients whenever jkpug was interacted with or add/remove was used. i don't think status changes are a particularly good/uniform way to determine activity.

**Comment by ensiform** [comment]
Yeah but should someone be allowed to idle only to have a gather start and they aren't even there?

**Comment by Razish** [comment]
That's what I meant by "refresh the list of pending clients whenever jkpug was interacted with or add/remove was used" - it would clean out "old" players / timeouts which is why you can `!add` again to "renew" your subscription to that gametype.
So someone can add, go do something that takes 15 minutes (food + coffee etc) then come back and hopefully it's about to start.
The timeout can/should be configurable.
